docs/architecture/psa-migration/md-cipher-dispatch.md

```diff
@@ -477,3 +503,57 @@ The architecture can be extended to support `MBEDTLS_PSA_CRYPTO_CLIENT` with a l
 
 * Compile-time dependencies: instead of checking `defined(MBEDTLS_PSA_CRYPTO_C)`, check `defined(MBEDTLS_PSA_CRYPTO_C) || defined(MBEDTLS_PSA_CRYPTO_CLIENT)`.
 * Implementers of `MBEDTLS_PSA_CRYPTO_CLIENT` will need to provide `psa_can_do_hash()` (or a more general function `psa_can_do`) alongside `psa_crypto_init()`. Note that at this point, it will become a public interface, hence we won't be able to change it at a whim.
+
+### Cipher light
```

Comment:
This document should include some notes about testing. That was missing for md — and thanks for doing that as part of the project.

I think the test strategy for md worked well and can be repeated for cipher. We need to be a bit more precise because ciphers are more complex: with hashes it's just whether an algorithm is disabled, accelerated-only, accelerated+builtin or builtin-only. With ciphers, we have to consider the block cipher and the mode. It's not as bad as ECC light, but still warrants a short section.

I see two aspects to cover: making sure we're testing the right configurations, and making sure that the configurations do what we want. For the second point, driver parity testing is a key component (that was a good idea!) and I see no reason to doubt that it'll work as well for cipher as it has for md and ECC. For the first point, we need to review the cipher-related components of `all.sh`, which puts at least https://github.com/Mbed-TLS/mbedtls/issues/8315 on the critical path.